.mypy.ini

```diff
@@ -1,4 +1,5 @@
 [mypy]
+plugins = pydantic.mypy
```

Comment:
The Pydantic plugin is already specified on line 11 of this file

Reply:
Going to remove, this extra line fixes pre-commit issues with mypy for python 3.9 but causes other issues on 3.11. Will remove and find an alternative way that works across all supported python versions.

Reply:
Removed, this will be fixed by pinning in .pre-commit-config.yaml